6-AdvancedScenarios/1-call-api-obo/AppCreationScripts/sample.json

```diff
@@ -5,115 +5,76 @@
         "Client": "React SPA",
         "Service": "Node.js web API",
         "RepositoryUrl": "ms-identity-javascript-react-tutorial",
-        "Endpoint": "AAD v2.0"
+        "Endpoint": "AAD v2.0",
+        "Platform": "javascript",
+        "Languages": [
+            "typescript",
+            "Node"
+        ],
+        "Description": "Handling Conditional Access challenges in an Azure AD protected Node.js web API calling another protected Node.js web API on behalf of a user",
+        "products": ["azure-active-directory", "msal-js", "passport-azure-ad"]
     },
     "AADApps": [
         {
-            "Id": "DownstreamAPI",
-            "Name": "msal-react-downstream",
-            "Kind": "WebApi",
-            "HomePage": "http://localhost:7000/api",
-            "Audience": "AzureADMyOrg",
-            "ManualSteps": [
-                {
-                    "Comment": "Navigate to the Manifest and set 'accessTokenAcceptedVersion' to '2' instead of 'null'"
-                },
-                {
-                    "Comment": "Create a new conditional access policy as described in the sample's README"
-                }
-            ]
-        },
-        {
-            "Id": "MiddletierAPI",
-            "Name": "msal-react-middletier",
+            "Id": "service",
+            "Name": "msal-react-api",
```

Comment:
> 
>  "Name": "msal-react-api",
> 

Can we rename this to something like msal-node-api

Reply:
Done!